src/app/android/app/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Trying to add comments in json files give me errors. I suppose we could ignore those errors, but that doesnt seem like good practice?

Reply:
If it bring a lot of value, consider making a MD file to talk about it.

Reply:
I already put what I wanted to say in the md file. The only reason I mention it here is because someone could not read the md file and it would have been nice to say right here that it's ok to put the firebase api key here.

Reply:
Should be fine then.

Reply:
@jeanplevesque

Reply:
yeah comments are not officially supported in the json format